_chapters/02_category.md

```diff
@@ -283,11 +283,11 @@ So, we can say that what we call "elements" of a set are the functions from the
 Defining the singleton set using functions
 ---
 
-Now, as we came up with some definition of set *element*, using just functions, we can try to draw the elements of our set as an external diagram.
+Now that we have come up with a definition of a set *element* using just functions, we can try to draw the elements of our set as an external diagram.
 
 ![Functions from the singleton set](../02_category/elements_singleton_functions_partly_external.svg)
 
-However, our diagram is not yet fully external, as it depends on the idea of the singleton set, i.e. the set with one *element*. Furthermore, this makes the whole definition circular, as we have to already have the concept of an element in order to define the concept of an one-element set. 
+However, our diagram is not yet fully external, as it depends on the idea of the singleton set, i.e. the set with one *element*. Furthermore, this makes the whole definition circular, as we have to already define the concept of an element in order to derive the concept of an one-element set.
```

Comment:
I don't use "derive" in the book, and I am trying to reduce the number of terms used.